src/components/DrawerTabPanels/DetailsTabPanel/DetailsTabPanel.tsx

```diff
@@ -26,6 +27,7 @@ const mapDispatchToProps = (dispatch) => ({
   setHoverWindowMode: (value) => dispatch(setHoverWindowMode(value)),
   setAggregation: (value) => dispatch(selectVectors(value, false)),
   setHoverStateOrientation: (value) => dispatch(setHoverStateOrientation(value)),
+  setGenericFingerprintAttributes: (value) => dispatch(setGenericFingerprintAttributes(value)),
```

Comment:
we want to get rid of the old syntax, so new features should use the hooks:
useDispatch to dispatch actions and usePSESelector to select specific pieces of the store